src/components/checkbox/examples/checkbox-helper-text.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, h } from '@stencil/core';
+
+/**
+ * With `helperText`
+ *
+ * Checkboxes can have a helper text, which is useful when providing additional information
+ * can clarify functionality of the checkbox for the user.
+ *
+ * The helper text is displayed when user hovers the checkbox, or focuses on it using keyboard
+ * navigation. However, on touchscreen devices, the helper text is always displayed.
+ */
+
+@Component({
+    tag: 'limel-example-checkbox-helper-text',
+    shadow: true,
+})
+export class CheckboxHelperTextExample {
+    public render() {
+        return (
+            <section>
+                <div>
+                    <limel-checkbox
+                        label="I accept trems of use"
```

Comment:
```suggestion
                        label="I accept terms of use"
```
🙂

Reply:
Otherwise great, so you can fix and squash that in one go, if you don't make any other changes, and I'll approve.

Reply:
thanks. I did it :)